src/demoServer.ts

```diff
@@ -66,7 +64,7 @@ app.use(bodyParser.urlencoded({ extended: true }));
 
 app.post('/ask', [logRequestMiddleware, askMiddleware, logResultsMiddleware]);
 
-const port = 80;
+const port = 1234;
```

Comment:
maybe let's use env var for this as well?

Reply:
I shouldn't have committed this change. Removed.